test/it/parser/src/main/resources/sql/supported/dml/select.xml

```diff
@@ -28,8 +28,8 @@
     <sql-case id="select_with_same_table_name_and_alias_column_with_owner" value="SELECT t_order.order_id,t_order.user_id,status FROM t_order t_order WHERE t_order.user_id = ? AND order_id = ?" db-types="MySQL,H2" />
     <sql-case id="select_not_equal_with_single_table" value="SELECT * FROM t_order_item WHERE item_id &lt;&gt; ? ORDER BY item_id" />
     <sql-case id="select_exclamation_equal_with_single_table" value="SELECT * FROM t_order_item WHERE item_id != ? ORDER BY item_id" />
-    <sql-case id="select_not_in_with_single_table" value="SELECT * FROM t_order_item WHERE item_id IS NOT NULL AND item_id NOT IN (?, ?) ORDER BY item_id" />
-    <sql-case id="select_not_between_with_single_table" value="SELECT * FROM t_order_item WHERE item_id IS NOT NULL AND item_id NOT BETWEEN ? AND ? ORDER BY item_id" />
+    <sql-case id="select_not_in_with_single_table" value="SELECT * FROM t_order_item WHERE item_id IS NOT NULL AND item_id NOT IN (?, ?) ORDER BY item_id" db-types="MySQL"/>
```

Comment:
Please do not change db-types here. This will reduce our test case.

Reply:
@strongduanmu Sir the reason for changing db-types is because without db-types, the test case is running for other database parsers like SQL92, sqlServer etc. and those files are not optimised for these QUERIES.
I added support for MySQL only. Should I add support for other database parsers too ? 
Thank you :)

Reply:
> @strongduanmu Sir the reason for changing db-types is because without db-types, the test case is running for other database parsers like SQL92, sqlServer etc. and those files are not optimised for these QUERIES hence Its giving error in InternalSQLParserIT file I added support for MySQL only. Should I add support for other database parsers too ? Thank you :)

@kanha-gupta, it's better to modify other sql parser logic together.

Reply:
okay